.travis.yml

```diff
@@ -1,194 +1,20 @@
-      compiler: gcc
-      script:
-        - set -e
-        - . tests/build-log-cflags-propagation.sh;
-        - export DISTCHECK_CONFIGURE_FLAGS="$CONFIGURE_FLAGS";
-        - exec_prop_check "make distcheck -j 3 V=1"
-      after_script:
-        - find . -name test-suite.log | xargs cat
-
-    - env: B=autotools-clang
-      dist: trusty
-      compiler: clang
-      script:
-        - set -e
-        - python2 -m pip install --user --cache-dir=$PWD/pip-cache pre-commit-hooks
-        - make --keep-going -j $(nproc) || make V=1 install;
-        - make install
-        - . scripts/get-libjvm-path.sh;
-        - export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$JNI_LIBDIR";
-        - make pytest-self-check
-        - make python-pep8
-        - make python-pylint
-        - make pytest-linters
-        - make VERBOSE=1 func-test
-        - make pytest-check
-
-    - env: B=style-check
-      dist: trusty
-      sudo: required
-      addons:
-      before_install:
-      - docker pull balabit/syslog-ng-devshell
-      install:
-      before_script:
-      script:
-      - set -e
-      - ./dbld/rules run-devshell RUN_COMMAND="cd /source && astyle --version && scripts/style-checker.sh format"
-      - git diff --exit-code
-
-    - env: B=copyright-check
-      sudo: false
-      compiler: gcc
-      install:
-      before_script:
-      script:
-        - set -e
-        - export COPYRIGHTVERBOSITY=1
-        - tests/copyright/check.sh . .
-
-    - env: B=commits-check
-      sudo: false
-      compiler: gcc
-      install:
-      before_script:
-      git:
-        submodules: false
-        depth: false
-      script:
-        - tests/commits/check.sh "${TRAVIS_COMMIT_RANGE}"
-
-    - env: B=cmake-submodules-and-pytest
-      compiler: gcc
-      dist: trusty
-      sudo: required
-      before_script:
-      git:
-        submodules: true
-      script:
-        - set -e
-        - python2 -m pip install --user --cache-dir=$PWD/pip-cache pre-commit-hooks
-        - mkdir build
-        - cd build
-        - cmake
-            -DCMAKE_C_FLAGS=-Werror
-            -DPYTHON_VERSION=2
-            -DCMAKE_INSTALL_PREFIX=$HOME/install/syslog-ng
-            ..
-        - make --keep-going -j $(nproc) all check install
-        - make pytest-self-check
-        - make python-pep8
-        - make python-pylint
-        - make pytest-linters
-        - make VERBOSE=1 func-test
-        - make pytest-check
-
-    - env: B=cmake
-      compiler: clang
-      dist: trusty
-      sudo: required
-      before_script:
-      script:
-        - set -e
-        - mkdir build
-        - cd build
-        - cmake
-            -G Ninja
-            -DCMAKE_C_FLAGS=-Werror
-            -DPYTHON_VERSION=2
-            -DCMAKE_INSTALL_PREFIX=$HOME/install/syslog-ng
-            ..
-        - ninja all check install
-
     - env: B=cmake-arm
       compiler: gcc
       sudo: required
       arch: arm64
       addons:
         apt:
+          sources:
```

Comment:
IIRC every package listed below actually available in the default repository, and so it is not needed to include this repository. Specially due to this specific repo url do not provide arm based package, that supplied via raspberry path.

Reply:
Thanks, fixed :)